src/transport/CryptoContext.h

```diff
@@ -114,19 +129,13 @@ class DLL_EXPORT CryptoContext
     CHIP_ERROR Decrypt(const uint8_t * input, size_t input_length, uint8_t * output, const PacketHeader & header,
                        const MessageAuthenticationCode & mac) const;
 
-    ByteSpan GetAttestationChallenge() const { return ByteSpan(mKeys[kAttestationChallengeKey], Crypto::kAES_CCM128_Key_Length); }
-
-    /**
-     * @brief
-     *   Memory overhead of encrypting data. The overhead is independent of size of
-     *   the data being encrypted. The extra space is used for storing the common header.
-     *
-     * @return number of bytes.
-     */
-    size_t EncryptionOverhead();
+    ByteSpan GetAttestationChallenge() const
+    {
+        return ByteSpan(mKeys[kAttestationChallengeKey].data(), mKeys[kAttestationChallengeKey].size());
+    }
 
 private:
-    typedef uint8_t CryptoKey[Crypto::kAES_CCM128_Key_Length];
+    using CryptoKey = std::array<uint8_t, Crypto::kAES_CCM128_Key_Length>;
```

Comment:
We should not store the keys themselves for serialization. We should store the sharedSecret derived during PASE/CASE. That way we can regenerate the keys with HDKF and store less stuff. The sharedSecret is needed for resumption anyway.